src/Symfony/Component/Form/Tests/Extension/Core/Type/BirthdayTypeTest.php

```diff
@@ -11,14 +11,15 @@
 
 namespace Symfony\Component\Form\Tests\Extension\Core\Type;
 
+use Symfony\Component\Form\Extension\Core\Type\BirthdayType;
 use Symfony\Component\OptionsResolver\Exception\InvalidOptionsException;
 
 /**
  * @author Stepan Anchugov <kixxx1@gmail.com>
  */
 class BirthdayTypeTest extends DateTypeTest
 {
-    public const TESTED_TYPE = 'Symfony\Component\Form\Extension\Core\Type\BirthdayType';
+    public const TESTED_TYPE = BirthdayType::class;
```

Comment:
Is there a reason for using a const?

Nowadays I would expect an abstract method I have to implement, but not sure it's worth to deprecate and change that behavior

Reply:
> Nowadays I would expect an abstract method I have to implement

That's exactly what I thought when doing the change. I think that this change worth it for all the benefits there are to use `::class` (better analysis, resilient to renaming, etc.), and also think that in a second time, that would be nice to indeed think of maybe replacing this with an abstract method. Definitely :+1: on this idea. This is what's done in Validator (kind-of).

Reply:
Especially since the "getTestedType()" method already exists... :thinking:

Reply:
Sorry, I objected this suggestion to keep the diff small, see discussion in the PR.